stubs/fpdf2/fpdf/enums.pyi

```diff
@@ -0,0 +1,171 @@
+from _typeshed import Incomplete, Self
+from enum import Enum, IntEnum
+
+from .syntax import Name
+
+class DocumentState(IntEnum):
+    UNINITIALIZED: int
+    READY: int
+    GENERATING_PAGE: int
+    CLOSED: int
+
+class CoerciveEnum(Enum):
+    @classmethod
+    def coerce(cls: type[Self], value: Self | str) -> Self: ...
+
+class CoerciveIntEnum(IntEnum):
+    @classmethod
+    def coerce(cls: type[Self], value: Self | str | int) -> Self: ...
+
+class Align(CoerciveEnum):
+    C: str
+    X: str
+    L: str
+    R: str
+    J: str
+
+class RenderStyle(CoerciveEnum):
+    D: str
+    F: str
+    DF: str
+    @property
+    def operator(self): ...
+    @property
+    def is_draw(self): ...
+    @property
+    def is_fill(self): ...
+    @classmethod
+    def coerce(cls, value): ...
```

Comment:
This looks to me like it has the same signature as the superclass, I think we can delete it here